### PR TITLE
Increase fluent-bit resources

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**How to categorize this PR?**

  /area logging
  /kind enhancement

**What this PR does / why we need it**:
Increase `fluent-bit` resources because of OOM issue on Dev

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:
@vlvasilev @ialidzhikov @vpnachev 

**Release note**:

```improvement operator
None
```

### CODE REVIEW

**Comment by rfranzke** [comment]
Wouldn't defining a VPA object make more sense?

**Comment by Kristian-ZH** [comment]
> Wouldn't defining a VPA object make more sense?

I was thinking on it too but I am not sure how exactly the VPA is working (fluent-bit has in-memory buffer with logs and if it is getting scaled very often it will be a problem) so maybe we will discuss it with @vlvasilev next week cuz this week we are both in vacation :D

**Comment by rfranzke** [comment]
OK, thanks, then we start with this PR/the static limit increase and continue to work on a dynamic solution as soon as you're back. Enjoy your vacation!

/lgtm

File: charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml [hunk 1]
```diff
@@ -35,11 +35,11 @@ spec:
...
           limits:
-            cpu: 100m
-            memory: 100Mi
+            cpu: 300m
+            memory: 400Mi
           requests:
-            cpu: 20m
-            memory: 40Mi
+            cpu: 200m
+            memory: 200Mi
```
Comment (vpnachev) [inline charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml:42]: I am fine with the increased limits, but why are you increasing the requests to so high values?

Comment (rfranzke) [reply charts/seed-bootstrap/charts/fluent-bit/templates/fluent-bit-daemonset.yaml:42]: Hm, indeed, it seems not necessary. The old request values were quite OK

**Comment by rfranzke** [comment]
As @Kristian-ZH is on vacation I'll 
/close this PR and open a new one that will also fix https://github.com/gardener/gardener/issues/2650

**Pull request closed without merging**